lib/fog/libvirt/requests/compute/list_volumes.rb

```diff
@@ -33,7 +39,7 @@ def volume_to_attributes(vol)
                 :allocation  => bytes_to_gb(vol.info.allocation),
                 :capacity    => bytes_to_gb(vol.info.capacity),
               }
-          rescue Libvirt::RetrieveError
+          rescue ::Libvirt::RetrieveError, ::Libvirt::Error
```

Comment:
I am not a Ruby expert, but I had problems with the change from #14. I was getting the following error:
```
..../gems/fog-libvirt-0.0.3/lib/fog/libvirt/requests/compute/list_volumes.rb:40:in `rescue in volume_to_attributes': uninitialized constant Fog::Compute::Libvirt::RetrieveError (NameError) (RuntimeError)
```

The actual exception being thrown is `Libvirt::RetreiveError`, so `Fog::Compute::` is not correct. I prepended the exception with `::` to get it working.